frontend/third_party/argo-ui/argo_template.ts

```diff
@@ -61,15 +48,12 @@ export interface Artifact {
    * Path is the container path to the artifact
    */
   path?: string;
-  /**
-   * Raw contains raw artifact location details
-   */
-  raw?: RawArtifact;
```

Comment:
Are you sure we want to remove this? The KFPv1 compiler uses raw artifacts.

Reply:
+1, also it's OK to keep fields we do not use

Reply:
Argo removed it, which is strange...

Looks like at least KFP UI is not using it.

Reply:
It seems to still exist in Argo:
https://github.com/argoproj/argo-workflows/blob/be44ce9ddce219026292317d42c977486f10c43f/pkg/apis/workflow/v1alpha1/workflow_types.go#L796
https://github.com/argoproj/argo-workflows/blob/be44ce9ddce219026292317d42c977486f10c43f/pkg/apis/workflow/v1alpha1/workflow_types.go#L915

This artifact type is used when there is a need for a constant artifact: when a constant value is passed to a component which consumes it as a file.